website/docs/reference/node-selection/methods.md

```diff
@@ -249,4 +249,30 @@ The `group` method is used to select models defined within a group.
   ```
 
 
+</VersionBlock>
+
+### The "wildcard" method
+<VersionBlock lastVersion="1.4">
+
+[Supported in v1.5 or newer.](/reference/node-selection/methods?version=1.5#the-wildcard-method)
```

Comment:
I tried to play with it to make it open in the same tab and broke it. We could use html to make it work but it might not be worth it. 
```suggestion
Supported in v1.5 or newer.
```